src/package-resolver.js

```diff
@@ -388,14 +388,18 @@ export default class PackageResolver {
       }
     }
 
+    if (manifest && getExoticResolver(version)) {
+      this.exoticRangeMatch(patterns.map(this.getStrictResolvedPattern.bind(this)), manifest);
```

Comment:
Well, this means the tests don't cover this line :( we should add a test case in a follow-up.